enterprise-suite/scripts/lbc.py

```diff
@@ -539,7 +547,14 @@ def install(creds_file):
             chart_file = args.local_chart
         else:
             chart_name = "{}/{}".format(args.repo_name, args.chart)
-            execute('helm repo add {} {}'.format(args.repo_name, args.repo))
+
+            # helm 3.3.2+ requires helm repo add --force-update to avoid errors re-running this script
+            supports_force_flag = version_is_at_least('helm version --client --short', '3.3.2')
+            if (supports_force_flag):
+                execute('helm repo add --force-update {} {}'.format(args.repo_name, args.repo))
+            else:
+                execute('helm repo add {} {}'.format(args.repo_name, args.repo))
```

Comment:
I might go with something like:
```
            force-flag = ""
            if (supports_force_flag):
                force-flag = "--force-update"
            execute('helm repo add {} {} {}'.format(force-flag, args.repo_name, args.repo))
```
but I'm not a python dude so maybe that's anathema.  I just like to see the command defined once in case it ever needs to get updated.

Reply:
Yeah, I've considered that too while writing the code, but decide that it would be clear to have it the way it is now. No strong opinion about it, though, and I would be happy with whichever option.

Reply:
No strong opinions here either tbh, I agree that one command would be preferable but I think 2 is OK, if we need more variants then I'd refactor it out into generating the args dynamically.

Reply:
See 748fb2d. It looks clearer, indeed. :-)

Reply:
And I like your clever handling of the added space character!